example/sriov-network/sirov-network-node-policy-legacy.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: example-sriov-node-policy-legacy
+  namespace: network-operator
+spec:
+  resourceName: nvidia.com/sriov-legacy
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 1
+  nicSelector:
+    vendor: "158b"
+    deviceID: "158b"
```

Comment:
what is this deviceID= 158b?

Reply:
We need to specify deviceID from this list: https://github.com/Mellanox/network-operator/blob/d646d7ebca753baa69011afceefe7d9b2608e8f1/deployment/network-operator/charts/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies_crd.yaml#L81

Reply:
sorry I mean that can remove it and keep just he vendor

Reply:
ok, let's keep deviceID line commented as an example